### DISCUSSION TITLE
OMG. The new feed is so useless.

### DISCUSSION THREAD

**Discussion by tidwall** [community/community#66181] [Other Feature Feedback, Questions, & Ideas]
Please please. Don't turn my feed into a social network. 

I do not care about 99.9% of the stuff other people do.

All I care about are my repositories and the people I follow. 

Stuff I care about:

- When someone stars or forks one of my projects.
- When someone follows me.
- When someone files an issue in one of my projects.
- When someone opens a PR in one of my projects.
- When someone discusses an issue that I'm engaged with.
- **And** all of those things for projects or people that I **explicitly** follow.

Other than showing me my new followers, the new feed shows none of the other stuff I want.

This is the worst Github update ever.

**Comment by koppor** [comment]
Workaround: Use https://github.com/Gerrit0/old-github-feed

**Comment by bbbbx** [comment]
> Please please. Don't turn my feed into a social network.
> 
> I do not care about 99.9% of the stuff other people do.
> 
> All I care about are my repositories and the people I follow.
> 
> Stuff I care about:
> 
> * When someone stars or forks one of my projects.
> * When someone follows me.
> * When someone files an issue in one of my projects.
> * When someone opens a PR in one of my projects.
> * When someone discusses an issue that I'm engaged with.
> * **And** all of those things for projects or people that I **explicitly** follow.
> 
> Other than showing me my new followers, the new feed shows none of the other stuff I want.
> 
> This is the worst Github update ever.

100% agree

Twitter today, Github tomorrow. Github is being next "X"
